go-runtime/compile/build-template/_ftl.tmpl/go/main/go.mod.tmpl

```diff
@@ -1,3 +1,7 @@
 module main
 
-go {{ .GoVersion }}
\ No newline at end of file
+go {{ .GoVersion }}
+
+{{ if ne .FTLVersion "" }}
```

Comment:
For future reference, I think you can just do `{{ if .FTLVersion }}` to check for non-zero values.